### PR TITLE
Add arsentd_lev dataset

### PR DESCRIPTION
**Pull request by moussaKam**
Add The Arabic Sentiment Twitter Dataset for Levantine dialect (ArSenTD-LEV)

Paper: [ArSentD-LEV: A Multi-Topic Corpus for Target-based Sentiment Analysis in Arabic Levantine Tweets](https://arxiv.org/abs/1906.01830)
Homepage: http://oma-project.com/

### CODE REVIEW

File: datasets/arsentd_lev/arsentd_lev.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+class ArsentdLev(datasets.GeneratorBasedBuilder):
+    """"ArSenTD-Lev Dataset"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features({feature: datasets.Value("string") for feature in _FEATURES}),
```
Comment (lhoestq) [inline datasets/arsentd_lev/arsentd_lev.py:51]: Could you use the `ClassLabel` feature type for `Topic`, `Sentiment` and `Sentiment_Expression` ?
for example
```
"Topic": datasets.ClassLabel(names=["personal", 'sports', ...])
```

after this change you'll need to regenerate the dataset_infos.json file

File: datasets/arsentd_lev/README.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K"
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- sentiment-classification
```
Comment (lhoestq) [inline datasets/arsentd_lev/README.md:20]:
```suggestion
- sentiment-classification
- topic-classification
```
looks like you can also do topic classification with this dataset

**Review by lhoestq — COMMENTED** [review]
Very clean, thanks for adding this one !

I left a few comments.
In particular one suggestion is to use the `ClassLabel` feature type

File: datasets/arsentd_lev/arsentd_lev.py (hunk 1 above)
Comment (moussaKam) [reply datasets/arsentd_lev/arsentd_lev.py]: Done. 
As for Topic, there are some weird labels that are not supposed to be there. So I kept the `datasets.Value("string")`.

**Comment by lhoestq** [comment]
thanks ! can you also regenerate the dataset_infos.json file please ?

**Review by lhoestq — APPROVED** [review]
Thank you !

**Pull request merged**